vm-setup/roles/v1aX_integration_test/vars/main.yml

```diff
@@ -18,6 +18,7 @@ BMOPATH: "{{ lookup('env', 'BMOPATH') }}"
 IRONIC_DATA_DIR: "{{ lookup('env', 'IRONIC_DATA_DIR') }}"
 KUBECONFIG_PATH: "/home/ubuntu/.kube/config"
 KUBERNETES_VERSION: "{{ lookup('env', 'KUBERNETES_VERSION') | default('v1.20.4', true) }}"
+TARGET_KUBECONFIG: "/tmp/kubeconfig-{{ CLUSTER_NAME }}.yaml"
```

Comment:
Ow, I guess this answers my previous two questions right? So this value is always fixed? no defaults?

Reply:
Yep, that's right. It is hardcoded once here instead of several times within the tasks.